byoqm/source_coordinator/query_translations.py

```diff
@@ -0,0 +1,145 @@
+query_lang = {
+    "python": {
+        "parameters": "parameters: (parameters)",
+        "bool_operator": "condition: (boolean_operator) right: (boolean_operator)",
+        "bool_operator_child": "boolean_operator",
+        "comment": "(comment)",
+        "function": "(function_definition)",
+        "function_block": "(function_definition body: (block) @function.block)",
+        "return": "(return_statement)",
+        "nested_controlflow1": """
+                                (module [
+                                (if_statement 
+                                    consequence: (block) @cons
+                                        )
+                                (if_statement 
+                                    consequence: (block) @cons
+                                    alternative: (_ [body: (block) consequence: (block) ] @cons) 
+                                        )
+                                (while_statement body: (block) @cons)
+                                (for_statement body: (block) @cons)]
+                                )
+
+                                (function_definition
+                                body: (block [
+                                    (if_statement 
+                                        consequence: (block) @cons
+                                            )
+                                    (if_statement 
+                                        consequence: (block) @cons
+                                        alternative: (_ [body: (block) consequence: (block)] @cons)
+                                            )
+                                    (while_statement body: (block) @cons)
+                                    (for_statement body: (block) @cons)])
+                                )
+                                """,
+        "nested_controlflow2": """
+                                (_ [
+                                (if_statement 
+                                    consequence: (block) @cons
+                                        )
+                                (if_statement 
+                                    consequence: (block) @cons
+                                    alternative: (_ [body: (block) consequence: (block) ] @cons) 
+                                        )
+                                (while_statement body: (block) @cons)
+                                (for_statement body: (block) @cons)]
+                                )
+                                """,
```

Comment:
`nestedcontrolflow1` and `nestedcontrolflow2` is a yikes. Not sure what a better solution is, but this is really unfortunate.

Reply:
I would be happy to look at it a second time tomorrow and see if we can do something smarter. However, I do not think we should spend that much more time on nested control flows, unless there is nothing better to do